controllers/Users.php

```diff
@@ -1,6 +1,7 @@
 <?php namespace RainLab\User\Controllers;
 
 use Auth;
+use Illuminate\Support\Facades\Redirect;
```

Comment:
```suggestion
use Redirect;
```
Use global facades please, that helps us prevent breaking changes whenever Laravel decides to make them because then it's easier for us to replace their facades with our own non-breaking ones. It's happened quite a few times already.

Reply:
Apologies, I should have caught that one.